source/runner/Executor.ts

```diff
@@ -161,7 +172,6 @@ export class Executor {
         const are = fails.length === 1 ? "is" : "are"
         const message = chalk.underline.red("Failing the build")
         output = `Danger: ${cross} ${message}, there ${are} ${fails.length} fail${s}.`
-        process.exitCode = 1
```

Comment:
I removed this because now this is more generic and is handled by the `strict` option, if this behaviour has to be kept I can add it on the line 143, by adding `this.options.stdoutOnly || this.options.jsonOnly || (this.ciSource && this.ciSource.useEventDSL` to the `if`.
I think is wrong here in any case because this method is called also by `handleResultsPostingToPlatform` just if `verbose` is true, then the build fails if is verbose as side effect also if you are running `danger ci`